.devcontainer/Dockerfile

```diff
@@ -206,74 +207,4 @@ RUN GOBIN="/usr/bin/" go install -v golang.org/x/tools/gopls@v0.8.3
 #### Update shared library configuration
 RUN ldconfig -v
 
-
```

Comment:
Instead of the removed block, we will use the Python environment built by Bazel. This requires us to build that environment initially - I'll update https://github.com/magma/magma/wiki/Contributing-Code-with-VSCode and https://github.com/magma/magma/wiki/Contributing-Code-with-IntelliJ once this gets merged.

While having to build the Python environment is inconvenient, this reduces the devcontainer download size, which is especially useful for people not working on the Python code anyway.